packages/cerebral/src/Controller.js

```diff
@@ -24,13 +24,21 @@ class Controller extends EventEmitter {
     this.componentDependencyStore = new DependencyStore()
     this.options = options
     this.flush = this.flush.bind(this)
-    this.devtools = devtools
-    this.model = new Model({}, this.devtools)
+
+    // Only allow devtools in development
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
Exactly. Now it's leaved user to decide condition on when to include devtools. Proposed change will not leave user a choice to have devtools in production build.